test/integration/k8s_test.go

```diff
@@ -184,10 +184,34 @@ func TestPodScaling(t *testing.T) {
 				}
 				t.Log("all pods have been allocated IPs")
 			}) {
-				errors.New("Pods don't have IP's")
-				return
+				t.Error("some pods don't have IPs")
 			}
 
+			t.Run("nodenetworkconfig has converged", func(t *testing.T) {
+				clusterCheckCtx, cancel := context.WithTimeout(ctx, 20*time.Minute)
+				defer cancel()
+				nnccli, err := nodenetworkconfig.NewClient(restConfig)
+				if err != nil {
+					t.Fatal(err)
```

Comment:
Not sure how descriptive the errors are here, would it be worth editorializing them a bit?